encrypted-media.js

```diff
@@ -288,6 +289,7 @@
     'media-resource': { func: videoref_helper, fragment: 'media-resource', link_text: 'media resource',  },
     'media-crossorigin': { func: code_videoref_helper, fragment: 'element-attrdef-media-crossorigin', link_text: 'crossorigin',  },
     'mime-types': { func: videoref_helper, fragment: 'mime-types', link_text: 'MIME types',  },
+    'valid-mime-type': { func: infrastructure_helper, fragment: 'valid-mime-type', link_text: 'valid media MIME type',  },
```

Comment:
Should the link text be "valid MIME type" and the link go to HTML5 or elsewhere ?